manim/scene/scene_file_writer.py

```diff
@@ -106,6 +100,32 @@ def init_output_directories(self, scene_name):
                 )
             )
 
+    def init_save_image(self, scene_name):
+        """Save images to the output file directory or the path of the scene. 
+        It does this after automatically detecting no animations are present and 
+        making the output an image instead of a video. 
+        """
+        if config["output_file"]:
```

Comment:
See my comment above: this duplicates part of the code from `init_output_directories`, that should be avoided.

As mentioned above, I think you can simply call `init_output_directories` above and revert this custom function here.